pyjexl/jexl.py

```diff
@@ -71,19 +72,20 @@ def wrapper(func):
     def parse(self, expression):
         try:
             return Parser(self.config).visit(self.grammar.parse(expression))
-        except ParsimoniousParseError as err:
-            raise ParseError('Could not parse expression: ' + expression) from err
+        except ParsimoniousParseError, err:
```

Comment:
Ah, no. The `from` bit should be removed, but you also removed the `as` keyword as well, which actually works in both versions and is the preferred syntax over separating the exception type from the name it's bound to. Just changing this line back to `except ParsimoniousParseError as err:` while leaving the `raise` line below it as you've changed it is fine.